tests/Billing/PaymentCardClientTest.php

```diff
@@ -50,8 +51,8 @@ public function get_payment_cards_by_id()
         $this->assertEquals("M44 0AB", $paymentCard->postcode);
         $this->assertEquals("4111111111111111", $paymentCard->cardNumber);
         $this->assertEquals("Visa", $paymentCard->cardType);
-        $this->assertEquals('2019-10-31T00:00:00+0000', $paymentCard->validFrom->format(DateTime::ISO8601));
-        $this->assertEquals('2035-10-31T00:00:00+0000', $paymentCard->expiry->format(DateTime::ISO8601));
+        $this->assertEquals('2019-10-31T00:00:00+0000', (new DateTime($paymentCard->validFrom))->format(DateTime::ISO8601));
```

Comment:
Wait, should this be in the format ISO8601 or `m-y`?

Reply:
Should be m/y will resolve shortly.